### PR TITLE
Various moving flow fixes

### PR DESCRIPTION
**Pull request by hugokallstrom**
Please check commit messages to see what fixes this PR contains

### Checklist

- [ ] Functionality is covered by an integration test
- [ ] Functionality is accessible in engineering mode

### CODE REVIEW

File: app/src/main/java/com/hedvig/app/feature/home/ui/HomeFragment.kt [hunk 1]
```diff
@@ -157,7 +158,7 @@ class HomeFragment : Fragment(R.layout.home_fragment) {
                     if (pendingAddress != null && pendingAddress.isNotBlank()) {
                         add(HomeModel.PendingAddressChange(pendingAddress))
                     }
-                    if (FeatureFlag.MOVING_FLOW.enabled) {
+                    if (FeatureFlag.MOVING_FLOW.enabled && marketManager.market == Market.SE) {
```
Comment (hedvigoscar) [inline app/src/main/java/com/hedvig/app/feature/home/ui/HomeFragment.kt:161]: Should we move this logic up to the feature flag itself? Now we're spreading it out in various places in the code, right? 🤔

File: app/src/main/res/layout/activity_checkout.xml [hunk 2]
```diff
@@ -100,7 +100,7 @@
                 android:hint="@string/simple_sign_login.text_field.label"
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toBottomOf="@id/emailInputContainer"
-                app:placeholderText="DDMMYYXXXX">
+                app:placeholderText="@string/simple_sign_login.text_field.helper_text">
```
Comment (hedvigoscar) [inline app/src/main/res/layout/activity_checkout.xml:103]: 👏

**Review by hedvigoscar — APPROVED** [review]
Approved upon addressing question 🙂

File: app/src/main/java/com/hedvig/app/feature/home/ui/HomeFragment.kt (hunk 1 above)
Comment (hugokallstrom) [reply app/src/main/java/com/hedvig/app/feature/home/ui/HomeFragment.kt:161]: I was thinking about that, but the feature flag will be removed when we release moving flow and we want to keep the market logic - so thats why they are separated here.

Comment (hedvigoscar) [reply app/src/main/java/com/hedvig/app/feature/home/ui/HomeFragment.kt:161]: Perhaps we should keep the flag until we reach general availability of the feature? 🙂

**Pull request merged**